### ISSUE TITLE
Sort by multiple columns specifying the direction

### ISSUE THREAD

**Issue by alvarotrigo** [DataTables/DataTables#592]
As per [this question in stackoverflow](http://stackoverflow.com/questions/31248861/sort-by-two-columns-with-datatables-cannot-read-property-stype-of-undefined/31249145#31249145) it would be nice to have a way to sort by multiple columns specifying the direction.

For example. 
Whenever a user clicks in the 2nd column to sort it, always sort it ascending by the 1st column.

This is quite useful when having groups of rows that should be kept together based on some ID, group name... etc.

**Comment by DataTables** [comment]
Agreed - this would be a sensible feature to add. I'll look into it for a future version.

**Comment by mpryvkin** [comment]
Although, it's nice to have `orderType` to accept sort method as `[[0, 'asc'],[1, 'asc]]`, there is a workaround, please see [this answer on StackOverflow](http://stackoverflow.com/questions/31248861/sort-by-two-columns-with-datatables-cannot-read-property-stype-of-undefined/31249145)
